http_server/server.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+import os
+import signal
+import socket
+import string
+import sys
+from threading import Event
+
+import chardet
+
+from images_downloader import ImageDownloader
+
+exit_event = Event()
+BUFFER_VALUE = 16
+SOCKET_TIMEOUT = 0.1
+
+
+def signal_handler(_signal, _):
+    """
+    Устанавливает глобальную переменную Event в значение true
+    """
+    exit_event.set()
+
+
+def receive_connection(connect: socket, client_address: socket,
+                       path_to_download: string):
+    """
+    Получает url из установленного соединения и скачивает из неё все картинки
+    :param connect: сокет с установленным соединением
+    :param client_address: адрес клиента
+    :param path_to_download: путь куда будут скачаны картинки
+    """
+    while True:
+        try:
+            data = connect.recv(BUFFER_VALUE)
+            print(f'Connected: {client_address}')
+            if data:
```

Comment:
Тогда уж

```suggestion
            if not data:
                print(f'Нет данных от:{client_address}')
                break
                
            print(f'Данные получены от: {client_address}')
            ...
```

меньше блоков с отступом